client/packages/common/src/styles/theme.ts

```diff
@@ -215,6 +220,12 @@ export const themeOptions = {
       notInUse: '#b0b0b0',
       text: '#fff',
     },
+    vaccinationStatus: {
+      given: 'success.light',
+      notGiven: 'error.main',
+      pending: 'cceStatus.notInUse',
+      late: 'cceStatus.functioningButNeedsAttention',
+    },
```

Comment:
Thought it was easier to define these explicitly, since the CCE stuff is. Fortunately, we can refer to existing colours within this!

Reply:
The screenshots in the issue are from the designs...

Reply:
I think all the colours we need are already in the theme (success, error, secondary)? So can probably just use those directly in the vaccine card component, rather than another layer of indirection here...

Otherwise, I think something like `chip` in the theme would be better, and pull colours from there, so can be reused. But yes, happy not to refactor CCE stuff right now 🙏

Reply:
> Otherwise, I think something like `chip` in the theme would be better, and pull colours from there

Yup, will definitely do this as well as get some definitive badge colour patterns established when we do https://github.com/msupply-foundation/open-msupply/issues/4779